experiments/opencog/pattern_matcher_vqa/run.sh

```diff
@@ -1,12 +1,12 @@
 #!/usr/bin/env bash
 
 python3 pattern_matcher_vqa.py \
-    --questions /mnt/fileserver/shared/models/vqa_split_multidnn/questions.txt \
+    --questions ~/projects/data/questions.934.txt \
```

Comment:
I would propose adding a script to get all necessary data from shared storage as well.

Reply:
I added the script